circle.yml

```diff
@@ -50,6 +52,6 @@ test:
     - stack --stack-yaml=stack-8.2.1.yaml test -j 2  haskell-ide-engine :
         timeout: 14400
 
-    - stack --stack-yaml=stack-8.2.2.yaml clean
-    - stack --stack-yaml=stack-8.2.2.yaml test -j 2  haskell-ide-engine :
+    - stack --stack-yaml=stack-8.0.2.yaml clean
```

Comment:
This one should be 8.2.1 - we clean the previous build

Reply:
ok